README.md

```diff
@@ -33,17 +33,17 @@ learning frameworks.
 
```

Comment:
I suggest to split example code in Training-Time Compression section for PyTorch to Quantization-Aware-Training and the config based approach for another scenarios.

Reply:
Done, please check

Reply:
Looks good for me. @MaximProshin, what do you think?